components/invitation_modal/invitation_modal_confirm_step_row.jsx

```diff
@@ -55,7 +57,11 @@ export default class InvitationModalConfirmStepRow extends React.Component {
                     {guestBadge}
                 </div>
                 <div className='reason'>
-                    {invitation.reason}
+                    <FormattedMessage
```

Comment:
This is not a valid approach, you are using the dynamically generated reason value to fill the `defaultMessage` but you are using the same key all the time, what means that is going to translate it wrongly. Or you pass all the data without translated and use here to build the FormattedMessage, or you pass the final translated version.